app/src/main/java/org/wikipedia/analytics/eventplatform/EventPlatformClient.java

```diff
@@ -39,7 +39,7 @@
      * Inputs: network connection state on/off, connection state bad y/n?
      * Taken out of iOS client, but flag can be set on the request object to wait until connected to send
      */
-    private static boolean ENABLED = isOnline() && isEventLoggingEnabled();
```

Comment:
Not sure but what's the reason for removing this?

Reply:
This is per our discussion to make this check only just before we send the events @cooltey